src/components/feedback/emptyState/NotificationsEmptyState.tsx

```
@@ -0,0 +1,90 @@
+  size = "md",
+}: NotificationsEmptyStateProps) {
+  const illustrationSize = size === "sm" ? "h-20 w-20" : "h-28 w-28";
+
+  return (
+    <EmptyState
+      title={title}
+      description={description}
+      size={size}
+      illustration={
+        <div className={"flex items-center justify-center " + illustrationSize}>
+          <svg viewBox="0 0 200 200" className="h-full w-full">
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)